src/Sylius/Bundle/ApiBundle/spec/Provider/PathPrefixProviderSpec.php

```diff
@@ -79,6 +79,6 @@ function it_returns_shop_prefix_if_there_is_no_logged_in_user(UserContextInterfa
     {
         $userContext->getUser()->willReturn(null);
 
-        $this->getCurrentPrefix()->shouldReturn('shop');
+        $this->getCurrentPrefix()->shouldReturn('');
```

Comment:
I assume, that the name of this test should be adjusted. Also, I'm wondering if it changes the current behavior. Have you checked it?

Reply:
All behats are green :) In our api we have three types users `AdminUser`, `ShopUser`, `Visitor` and two prefix types `admin` , `shop`. We assumed that Visitor will be using only `shop` prefix, but what if someone adds new endpoint without prefix? This changes must be added to properly work with changes on `RouteNameResolver.php`. This provider is used in all requests on tests and all test are fine. We have only this test for it.